Update/kage_003.txt

```diff
@@ -255,10 +255,10 @@ void main()
 	if (GetGlobalFlag(GADVMode)) {
 		OutputLineAll("", NULL, Line_ContinueAfterTyping);
 		OutputLine(NULL, "南井警部の見解は保守的ではなく、もっともなことだ。この地域の刑事課に他県のガス災害、そして陰謀云々を語ったところで、職務としてできることは何もない。",
-				NULL, "<size=-2>Inspector Minai wasn't purposefully being dismissive, her opinions are completely valid. Just because I discuss the possiblity of a conspiracy surrounding the gas disaster another town, doesn't mean that they have to follow up on it, especially when they don't even work in that prefecture.", Line_Normal);
+				NULL, "<size=-2>Inspector Minai wasn't purposefully being dismissive, her opinions are completely valid. Just because I discuss the possiblity of a conspiracy surrounding the gas disaster in another town doesn't mean that they have to follow up on it, especially when they don't even work in that prefecture.", Line_Normal);
```

Comment:
```suggestion
				NULL, "<size=-2>Chie Inspector Minai wasn't purposefully being dismissive, her opinions are completely valid. Just because I discuss the possibility of a conspiracy surrounding the gas disaster in another town doesn't mean that they have to follow up on it, especially when they don't even work in that prefecture.", Line_Normal);
```